changelog_entry.yaml

```diff
@@ -0,0 +1,4 @@
+- bump: minor
```

Comment:
https://casetext.com/statute/michigan-compiled-laws/chapter-206-income-tax-act-of-1967/income-tax-act-of-1967/section-206522-determination-of-amount-of-claim-election-of-classification-in-which-to-make-claims-single-claimant-per-household-entitled-to-credit-totally-and-permanently-disabled-defined-computation-of-credit-by-senior-citizen-reduction-of-claim-tables-maximum-credit-total-credit-allowable-under-part-and-part-361-of-natural-resources-and-environmental-protection-act

https://casetext.com/statute/michigan-compiled-laws/chapter-206-income-tax-act-of-1967/income-tax-act-of-1967/section-206520-credit-for-property-taxes-on-homestead-credit-for-person-renting-or-leasing-homestead-credit-in-excess-of-tax-liability-due-assignment-of-claim-to-mortgagor-by-senior-citizen-for-rent-reduction-eligibility-to-claim-credit-on-property-rented-or-leased-as-credit-for-person-receiving-aid-to-families-with-dependent-children-state-family-assistance-or-state-disability-assistance-payments-reduction-of-credit-for-claimant-whose-household-income-exceeds-certain-amount-adjustment-credit-claimable-by-senior-citizen-limitations-rules-form-determining-qualification-to-claim-credit-after-move-to-different-rented-or-leased-homestead-reduction-of-claim-for-return-of-less-than-12-months-total-credit-allowed-by-section-and-mcl-206522-united-states-consumer-price-index-defined